### DISCUSSION TITLE
Switching ethers to viem: Metamask - RPC Error: [ethjs-query] while formatting outputs from RPC Issue

### DISCUSSION THREAD

**Discussion by johnpaulcas** [wevm/wagmi#2567] [Q&A]
We don't know causing this issue but we are encountering this intermittent, anyone with any idea? this is happening when we transition from etherjs to viem

 
![Screenshot 2023-06-19 at 11 15 56 AM](https://github.com/wagmi-dev/wagmi/assets/44362950/603eaeb0-476e-4378-b99e-c88c96a69320)

**Comment by jxom** [comment]
Seems like a rate limiting error based off the message in that error in the screenshot (40 requests per 1 second). Are you sending too many transactions in a short timeframe in your UI? A minimal reproduction via https://new.wagmi.sh would help a lot.

**Reply by johnpaulcas** [reply]
hi @jxom we are just sending 1 tx, but this request happened after our API call but not related to blockchain
